internal/testutils/mock/mock_openshift_client.go

```diff
@@ -43,6 +43,14 @@ func (c *OpenShiftClient) State(apiURL string, bearerToken string, namespace str
 	return c.IdleState, nil
 }
 
+// Reset deletes a pod and start a new one
+func (c *OpenShiftClient) Reset(apiURL string, bearerToken string, namespace string) (err error) {
```

Comment:
`err`  can be deleted.

Reply:
only an fyi ... go allows `foo(x, y, z, string)` if `x, y, z` are all `string`s.